fdbserver/BlobWorker.actor.cpp

```diff
@@ -234,8 +235,8 @@ struct BlobWorkerData : NonCopyable, ReferenceCounted<BlobWorkerData> {
 };
 
 namespace {
-bool isBlobFileEncryptionSupported() {
-	bool supported = SERVER_KNOBS->ENABLE_BLOB_GRANULE_ENCRYPTION && SERVER_KNOBS->BG_RANGE_SOURCE == "tenant";
+bool isBlobFileEncryptionSupported(Reference<AsyncVar<ServerDBInfo> const> dbInfo) {
```

Comment:
nit: Do we need this routine or it can be abstracted out within `isEncryptionOpSupported` ?

Reply:
Ah okay that's what you meant with the previous comment, yeah i think it makes sense